mongomock/aggregate.py

```diff
@@ -576,7 +577,14 @@ def _handle_string_operator(self, operator, values):
             'pipeline, it is currently not implemented  in Mongomock.' % operator)
 
     def _handle_date_operator(self, operator, values):
-        out_value = self.parse(values)
+        if isinstance(values, dict) and \
```

Comment:
I suggest you use a simpler set equality: `values.keys() == {'date', 'timezone'}`

Reply:
thank you.
I updated it as you suggested.